corpora/proverbs.dep

```diff
@@ -27,3 +28,979 @@
+# text = Канга - кан.
+# text[eng] = Blood for blood. 
+# label_cg = JNW_checked
+# label_dep = to_check
+"<Канга>"
+	"кан" n dat @obl #1->2
+"<кан>"
+	"кан" n nom @root #2->0
+"<.>"
+	"." sent @punct #3->2
+
+# sent_id = proverbs:35
+# author:id = Сыргабековна&Сабыровна:138
+# text = Дарак бир жерден көгөрөт.
+# text[eng] = A tree becomes green in one place.  (i.e., when it's stable it thrives)
+# label_cg = JNW_checked
+# label_dep = to_check
+"<Дарак>"
+	"дарак" n nom @nsubj #1->4
+"<бир>"
+	"бир" num @amod #2->3
+"<жерден>"
+	"жер" n abl @obl #3->4
+"<көгөрөт>"
+	"көгөр" v iv aor p3 sg @root #4->0
+"<.>"
+	"." sent @punct #5->4
+
+# sent_id = proverbs:36
+# author:id = Сыргабековна&Сабыровна:331
+# text = Миң чеченди бир дөөрүк жеңиптир.
+# text[eng] = One babbler beat a thousand orators.
+# label_cg = JNW_checked
+# label_dep = to_check
+"<Миң>"
+	"миң" num @nummod #1->2
+"<чеченди>"
+	"чечен" n acc @obj #2->5
+"<бир>"
+	"бир" num @nummod #3->4
+"<дөөрүк>"
+	"дөөрүк" n nom @nsubj #4->5
+"<жеңиптир>"
+	"жең" v tv ifi evid p3 sg @root #5->0
+"<.>"
+	"." sent @punct #6->5
+
+# sent_id = proverbs:37
+# author:id = Сыргабековна&Сабыровна:372
+# text = Өтпөс өмүр, сынбас темир болбойт.
+# text[eng] = There is no unending life, no unbreakable iron(/steel).
+# label_cg = JNW_checked
+# label_dep = to_check
+"<Өтпөс>"
+	"өт" v iv neg gpr_fut @amod #1->2
+"<өмүр>"
+	"өмүр" n nom @parataxis #2->6
+"<,>"
+	"," cm @punct #3->6
+"<сынбас>"
+	"сын" v iv neg gpr_fut @amod #4->5
+"<темир>"
+	"темир" n nom @nsubj #5->6
+"<болбойт>"
+	"бол" v iv neg aor p3 sg @root #6->0
+"<.>"
+	"." sent @punct #7->6
+
+# sent_id = proverbs:38
+# author:id = Сыргабековна&Сабыровна:239
+# text = Ийгиликтин эрте-кечи жок.
+# text[eng] = There is no "sooner or later" to fortune.
+# label_cg = JNW_checked
+# label_dep = to_check
+"<Ийгиликтин>"
+	"ийгилик" n gen @obl #1->3
+"<эрте-кечи>"
+	"эрте-кеч" adj subst px3sp nom @nsubj #2->3
+"<жок>"
+	"жок" adj @root #3->0
+		"э" cop aor p3 pl @cop #4->3
+"<.>"
+	"." sent @punct #5->3
+
+# sent_id = proverbs:39
+# author:id = Сыргабековна&Сабыровна:256
+# text = Калп ырыс кесет, кайгы өмүр кесет.
+# text[eng] = A lie cuts happiness, grief cuts life.
+# label_cg = JNW_checked
+# label_dep = to_check
+"<Калп>"
+	"калп" n nom @nsubj #1->3
+"<ырыс>"
+	"ырыс" n nom @obj #2->3
+"<кесет>"
+	"кес" v tv aor p3 sg @parataxis #3->7
+"<,>"
+	"," cm @punct #4->7
+"<кайгы>"
+	"кайгы" n acc-ind @nsubj #5->7
+"<өмүр>"
+	“өмур” n nom @obj #6->7
+"<кесет>"
+	"кес" v tv aor p3 sg @root #7->0
+"<.>"
+	"." sent @punct #8->7
+
+# sent_id = proverbs:40
+# author:id = Сыргабековна&Сабыровна:345
+# text = Ооруну жашырсаң, өлүм ашкере кылат.
+# text[eng] = If you hide sickness/pain, death will reveal itself.
+# label_cg = JNW_checked
+# label_dep = to_check
+"<Ооруну>"
+	"оору" n acc @obj #1->2
+"<жашырсаң>"
+	"жашыр" v tv gna_cond p2 sg @advcl #2->6
+"<,>"
+	"," cm @punct #3->6
+"<өлүм>"
+	"өлүм" n nom @nsubj #4->6
+"<ашкере>"
+	"ашкере" adj @amod #5->4
```

Comment:
`#5->4` is almost definitely wrong.  What were the options I proposed earlier?

Reply:
Earlier when?

Reply:
Sorry, I meant the different ways of treating this that I mentioned in a comment in the spreadsheet.